Fix errors on serialization of values that are not AnyValue in LogRecord attributes (#4808)
open-telemetry/opentelemetry-python: CHANGELOG.md

## Unreleased

> - `opentelemetry-api`: Convert objects of any type other than AnyValue which fixes invalid type at WSGI request attributes

Reviewer: ```suggestion
- `opentelemetry-api`: Convert objects of any type other than AnyValue in attributes to string to be exportable
```

- Add reset for the global configuration object, for testing purposes
  ([#636](https://github.com/open-telemetry/opentelemetry-python/pull/636))
- Add support for programmatic instrumentation

>   ([#569](https://github.com/open-telemetry/opentelemetry-python/pull/569))

Reviewer: ```suggestion
  ([#579](https://github.com/open-telemetry/opentelemetry-python/pull/579))
```
